forge-gui/res/cardsfolder/upcoming/fade_from_history.txt

```diff
@@ -0,0 +1,8 @@
+Name:Fade from History
+ManaCost:2 G G
+Types:Sorcery
+A:SP$ Token | Defined$ Player.controlsEnchantment,Player.controlsArtifact | TokenScript$ g_2_2_bear | SubAbility$ DBDestroy | SpellDescription$ Each player who controls an artifact or enchantment creates a 2/2 green Bear creature token. Then destroy all artifacts and enchantments.
```

Comment:
How does this StackDescription look when cast?

Reply:
SpellDescription